src/Xamarin.Android.Build.Tasks/Xamarin.Android.Common.targets

```diff
@@ -1616,7 +1616,7 @@ because xbuild doesn't support framework reference assemblies.
   </PrepareAbiItems>
 </Target>
 
-<Target Name="_GenerateEnvironmentFiles" DependsOnTargets="_ReadAndroidManifest">
+<Target Name="_GenerateEnvironmentFiles">
```

Comment:
So `_ReadAndroidManifest` does need to run before `_GenerateJavaStubs` I think, does that still happen?

Reply:
That part looks OK:
https://github.com/dotnet/android/blob/b56d68fe2a2c2dffa47eb7e41ae08a00fda86367/src/Xamarin.Android.Build.Tasks/Xamarin.Android.Common.targets#L1577-L1578

But we can see if CI is green, maybe it will find some other issue from this.

Reply:
Yeah, looks like maybe `_ReadAndroidManifest` doesn't run at all now, checking it.

Reply:
So, the places we had `_GenerateEnvironmentFiles` listed for `$(_PrepareBuildApkDependsOnTargets)` and `$(_CreateBaseApkDependsOnTargets)` need to change to `_ReadAndroidManifest`.

I think what we have now makes sense, and CI is green.